packer-plugin-sdk/net/configure_port.go

```diff
@@ -1,3 +1,5 @@
+// package net contains some helper wrapping functions for the http and net
```

Comment:
```suggestion
// Package net contains some helper wrapping functions for the http and net
```
 just adding an upper case to look like the other docs